wgpu-types/src/lib.rs

```diff
@@ -6322,13 +6322,16 @@ pub struct InstanceDescriptor {
     pub backends: Backends,
     /// Which DX12 shader compiler to use.
     pub dx12_shader_compiler: Dx12Compiler,
+    /// Force ANGLE to support GLES 3.1, even on nonconformant implementations like D3D11.
+    pub force_angle_gles31: bool,
```

Comment:
Could we generalize this to "force gles minor version", with valid values being 0, 1, or 2.

Reply:
I decided to remove the `force` naming, and make it a fully general-purpose thing like the DX12 compiler. I also made it its own enum to enforce the values.